### PR TITLE
Simplify IDs

### PR DESCRIPTION
**Pull request by toanju**
this moves cdpid and cdptid to typedefs

### CODE REVIEW

File: src/rofl/common/crofbase.cc [hunk 1]
```diff
@@ -743,8 +744,8 @@ void
 crofbase::handle_connect_refused(
 		crofdpt& dpt, crofconn& conn)
 {
-	journal.log(LOG_INFO, "connection refused: dptid=%s, auxid=%s, raddr=%s",
-			dpt.get_dptid().str().c_str(), conn.get_auxid().str().c_str(), conn.get_raddr().str().c_str());
+	journal.log(LOG_INFO, "connection refused: dptid=0x%" PRIx64 ", auxid=%s, raddr=%s",
```
Comment (DFritzsche) [inline src/rofl/common/crofbase.cc]: as we discussed, please check the output of `dptid=0x%" PRIx64 "` , also this is repeated below

File: src/rofl/common/crofbase.h [hunk 2]
```diff
@@ -452,8 +452,8 @@ class crofbase :
 	rofl::crofdpt&
 	add_dpt() {
 		AcquireReadWriteLock rwlock(rofdpts_rwlock);
-		uint64_t id = 0;
-		while (rofdpts.find(cdptid(id)) != rofdpts.end()) {
+		cdptid id = 0;
+		while (rofdpts.find(id) != rofdpts.end()) {
 			id++;
 		}
 		rofdpts[cdptid(id)] = new crofdpt(this, cdptid(id));
```
Comment (DFritzsche) [inline src/rofl/common/crofbase.h]: please update

Comment (toanju) [inline src/rofl/common/crofbase.h]: done

Comment (DFritzsche) [inline src/rofl/common/crofbase.h]: Well, the same applies to line 460 and 611/612. :)

Comment (toanju) [inline src/rofl/common/crofbase.h]: k, found some more

**Pull request merged**